Clarify Manual Loading of Environment Variables in Sandbox Environments (#7822)
aws-amplify/docs: src/pages/[platform]/deploy-and-host/fullstack-branching/secrets-and-vars/index.mdx

### Local environment

> When working on your local machine, you must manually load the sandbox's environment variables. First, add the environment variable in your `.env.local` file. Then, a library such as dotenvx can load the environment variables and reference them with `process.env`.

Reviewer: ```suggestion
When working on your local machine, you must manually load the sandbox's environment variables. First, add the environment variable in your `.env.local` file. Then, a library such as [`@dotenvx/dotenvx`](https://www.npmjs.com/package/@dotenvx/dotenvx) can load the environment variables, which you can then reference with `process.env`.
```

Author: Thanks for the suggestion.  I've added the link.
